<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrbiconAppClass</class>
 <widget class="QMainWindow" name="OrbiconAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OrbiconApp</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open_Image"/>
    <addaction name="action_Add_Images_to_Catalog"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Match"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open_Image">
   <property name="text">
    <string>&amp;Open Image...</string>
   </property>
  </action>
  <action name="action_Add_Images_to_Catalog">
   <property name="text">
    <string>&amp;Add Images to Catalog...</string>
   </property>
  </action>
  <action name="action_Match">
   <property name="text">
    <string>&amp;Match</string>
   </property>
   <property name="toolTip">
    <string>Match</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="orbicon_app.qrc"/>
 </resources>
 <connections/>
</ui>
